source/about/mattermost-v10-changelog.md

```diff
@@ -7,6 +7,87 @@ Support for Mattermost Server v9.11 [Extended Support Release](https://docs.matt
 - See the [changelog in progress](https://bit.ly/2nK3cVf) for details about the upcoming release.
 ```
 
+(release-v10.5-extended-support-release)=
+## Release v10.5 - [Extended Support Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release Day: Februrary 14, 2025**
+
+### Compatibility
+ - Updated minimum Safari version to 17.4+ and minimum Firefox version to 119+.
+
+### Important Upgrade Notes
+ - v10.5 introduces Property System Architecture schema migration. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for details.
+ - The Compliance Export system has been overhauled. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for details.
+ - The Mattermost server has stopped supporting manual plugin deployment. Plugins were deployed manually when an administrator or some deployment automation copies the contents of a plugin bundle into the server's working directory. If a manual or automated deployment workflow is still required, administrators can instead prepackage the plugin bundles. See more details in [this forum post](https://forum.mattermost.com/t/deprecation-notice-manual-plugin-deployment/21192).
+ - Mattermost has stopped official Mattermost server builds for the Microsoft Windows operating system. Administrators should migrate existing Mattermost server installations to use the official Linux builds. See more details in [this forum post](https://forum.mattermost.com/t/deprecation-notice-server-builds-for-microsoft-windows/21498).
+
+### Breaking Changes
+ - The internal workings of the ``PluginLinkComponent`` in the webapp have been improved. Plugins that register link tooltips using ``registerLinkTooltipComponent`` will see changes in how tooltip components are managed. Link tooltips are now unmounted from the DOM by default, significantly improving performance. They are only mounted when a link is hovered over or focused. As a result, plugins may need to update their components to properly handle mounting and unmounting scenarios.
```

Comment:
```suggestion
 - The internal workings of the ``PluginLinkComponent`` in the webapp have been changed to unmounted link tooltips from the DOM by default, significantly improving performance. Plugins that register link tooltips using ``registerLinkTooltipComponent`` will see changes in how tooltip components are managed. They are only mounted when a link is hovered over or focused. As a result, plugins may need to update their components to properly handle mounting and unmounting scenarios.
```

@M-ZubairAhmed can we be more specific on what plugin developers need to update here? We can possibly link the example of what was done in the GH plugin.

Reply:
thanks @marianunez added more information as expected above